scripts/obtain/dotnet-install.ps1

```diff
@@ -200,8 +204,10 @@ function GetHTTPResponse([Uri] $Uri)
             # Default timeout for HttpClient is 100s.  For a 50 MB download this assumes 500 KB/s average, any less will time out
             # 10 minutes allows it to work over much slower connections.
             $HttpClient.Timeout = New-TimeSpan -Minutes 10
-            $Response = $HttpClient.GetAsync($Uri).Result
+            $ActualUri = if (($Uri -like "$AzureFeed*") -or ($Uri -like "$UncachedFeed*")) { "${Uri}${FeedCredential}" } else { $Uri }
```

Comment:
Why are we checking the $Uri here? Why would we not append the FeedCredential?

Reply:
Today, it is always appended. My thinking was that may not always be true. Since `function GetHTTPResponse` is a general-purpose method, I wanted to avoid the possibility of appending a query string to a URL that shouldn't have the token added.

But I'd be okay to drop this if you're not worried about it.

Reply:
I think we drop the check now; even though it 'may not always be true'. When that situation arises, then the dev should put in an appropriate check.

Reply:
Sounds good. Updated the PR to drop this check.